test/fixtures/data.py

```diff
@@ -106,12 +106,42 @@ def cdms_companies(all_companies: DataFrame) -> DataFrame:
     df_cdms.columns = ["crn", "cdms"]
 
     df_cdms.reset_index(names="id", inplace=True)
-    df_cdms["id"] = df_cdms["id"].apply(lambda x: uuid.UUID(int=x))
+    df_cdms["id"] = df_cdms["id"].apply(lambda x: UUID(int=x))
     df_cdms = df_cdms.convert_dtypes(dtype_backend="pyarrow")
 
     return df_cdms
 
 
+@pytest.fixture(scope="session")
+def revolution_inc(
+    crn_companies: DataFrame, duns_companies: DataFrame, cdms_companies: DataFrame
+) -> dict[str, str]:
+    """
+    Revolution Inc. as it exists across all three datasets.
+
+    UUIDs are converted to strings to mirror how Matchbox stores them.
+    """
+    crn_ids = crn_companies[
+        crn_companies["company_name"].str.contains("Revolution", case=False)
+    ]["id"].tolist()
+
+    duns_ids = duns_companies[
+        duns_companies["company_name"].str.contains("Revolution", case=False)
+    ]["id"].tolist()
```

Comment:
the tests later (I think) asssume that there are multiple crn IDs and one duns ID. Can we call this out here? I also don't like that this is so tied to a large CSV shared by all tests. Might have to be part of later work on improving tests though